cam2image/src/cam2image.cpp

```diff
@@ -99,6 +99,7 @@ int main(int argc, char * argv[])
         convert_frame_to_message(flipped_frame, i, msg);
       }
       std::cout << "Publishing image #" << i << std::endl;
+      msg->header.frame_id = std::to_string(i);
```

Comment:
Why is this added here? That is already done in https://github.com/ros2/demos/blob/4f409e9e89796143a94b9fa1cd18988d64edb64f/cam2image/src/cam2image.cpp#L52